src/ecalc/libraries/libecalc/common/libecalc/core/consumers/compressor/component.py

```diff
@@ -90,20 +90,25 @@ def evaluate(
             else:
                 aggregated_result.extend(model_result)
 
+        energy_usage = TimeSeriesRate(
+            values=aggregated_result.energy_usage,
+            timesteps=evaluated_timesteps,
+            unit=aggregated_result.energy_usage_unit,
+            regularity=regularity,
+        )
+
+        if energy_usage.unit == Unit.STANDARD_CUBIC_METER_PER_DAY:
+            energy_usage = energy_usage.to_calendar_day()  # legacy consumer converts to calendar day if fuel consumer
```

Comment:
can you elaborate a bit more on the comment, I dont quite understand it as it is